src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctype/InputJavadocTypeAllowedAnnotations.java

```diff
@@ -1,6 +1,6 @@
+//non-compiled with javac: Compilable with Java9
```

Comment:
ones again, travis need to find files for jdk before jdk9 and compile them by jdk8.
We never drop files out of compilation check. If there is no such check it result in " code become in-compiled, our parser generate weird AST, weird code appear in Check."

Reply:
done